critiquebrainz/frontend/external/bookbrainz_db/edition_group.py

```diff
@@ -60,7 +60,10 @@ def fetch_multiple_edition_groups(bbids: List[uuid.UUID]) -> dict:
                     disambiguation,
                     identifier_set_id,
                     relationship_set_id,
-                    json_agg( acn ORDER BY "position" ASC ) as author_credits
+                    COALESCE( json_agg( acn ORDER BY "position" ASC )
```

Comment:
it's great that you were able to find coalesce!
I remember that we added some manual data for tests for relationships in BB entities. Would it also be possible to add some data with a null artist credit to ensure that this item isn't returned in the query?

Reply:
Yes, it is possible. I'll add tests for both cases if author credits exist and if they don't.